api.planx.uk/types.ts

```diff
@@ -12,3 +12,104 @@ export interface Flow {
     [key: string]: Node;
   };
 }
+
+export interface UserData {
+  answers?: Array<string>;
+  data?: Record<string, any>;
+  auto?: boolean;
+  feedback?: string;
+}
+
+export type Breadcrumb = Record<string, UserData>;
+
+export interface Team {
+  slug: string;
+  name: string;
+  domain?: string;
+  notifyPersonalisation: {
+    helpEmail: string;
+    helpPhone: string;
+    helpOpeningHours: string;
+    emailReplyToId: string;
+  };
+}
+
+export interface Passport {
+  data?: Record<string, any>;
+}
+
+export interface LowCalSession {
+  govUkPayment: any;
+  data: {
+    passport: Passport;
+    breadcrumbs: Breadcrumb;
+    govUkPayment: GovUKPayment;
+    id: string;
+  };
+  id: string;
+  created_at: string;
+  updated_at: string;
+  has_user_saved: boolean;
+  flow: {
+    slug: string;
+  };
+}
+
+export interface NotifyConfig {
+  personalisation: {
+    has_user_saved?: boolean | undefined;
```

Comment:
I think this is a mistake - `has_user_saved` is a column on the lowcal_sessions table whilst `NotifyConfig.personalisation` represents the variables that can be passed into an email sent via Notify.